src/core/rtkQuery/splitV3Api/fileApi.ts

```diff
@@ -31,11 +25,12 @@ const fileApi = splitV3Api.injectEndpoints({
     }),
 
     // Run a file through AVDump and return the result.
-    postFileAVDump: build.query<AVDumpResultType, number>({
+    postFileAVDump: build.mutation<void, number>({
       query: fileId => ({
-        url: `File/${fileId}/AVDump`,
+        url: `File/${fileId}/AVDump?immediate=false`,
         method: 'POST',
       }),
+      invalidatesTags: ['AVDumpEvent'],
```

Comment:
If you have set `immediate=false`, how is invalidation supposed to work?
`getFiles` query will run before dumping is complete

Reply:
Immediate false is to make it queue the item instead of running it then and there. It defulaults to true as to not break web ui v1. The tag invalidation is to get theblist to update with the new information that the command is queued (or running) for the file(s).

Reply:
The files api gives the information that the file is queued?

Reply:
every file has an avdump section sent with it now, and in it we get info about if the file has been successfully dumped before, if a command is session is queued or if a session is running. also if a session is running then additional data is available from the session.